cmake/GenerateBuiltins.cmake

```diff
@@ -400,7 +408,7 @@ function (generate_common_builtins resultList)
     message (STATUS "ISPC will be built with support of ${supported_oses} for ${supported_archs}")
     foreach (bit 32 64)
         foreach (os_name "windows" "linux" "freebsd" "macos" "android" "ios" "ps4" "web")
-            foreach (arch "x86" "arm" "wasm32")
+            foreach (arch "x86" "arm" "wasm32" "wasm64")
```

Comment:
`wasm32` here needs to be changed to `wasm`. Though `target_arch` in `builtin_to_cpp()` needs to be still `wasm32` and `wasm64`.

I.e. it needs to be similar to `x86` (which triggers `x86` and `x86_64`) and `arm` (which triggers `armv7` and `aarch64`).